### DISCUSSION TITLE
Cannot reinstall dshidmini and BthPS3 after reebooting pc

### DISCUSSION THREAD

**Discussion by PipoSovi** [nefarius/DsHidMini#273] [Support]
Hi,

Recently I had installed dshidmini and BthPS3 successfully. I had no problems running it with rpcs3.
However, due to unrelated issues, I rebooted my pc back to factory settings. Following that I tried to reinstall the drivers, however, this time nothing really happened.
After installing the file "dshidmini", my ds3 remote stopped blinking after the 3rd flash. Besides, when I opened the actual app, dshidmini could not recognize it. Once I removed the driver, the controller started blinking as usual.
Another weird this is when I try to install the BthPS3 , a popup message says that a previous version is already installed.

What can be done?
![image](https://user-images.githubusercontent.com/122615932/212327455-d1791891-beb7-41ee-b365-70a108aed01f.png)

**Comment by nefarius** [comment]
[Try this](https://github.com/nefarius/Legacinator/releases) to clean up and then try reinstall.
